### ISSUE TITLE
DPL-834 Add 007c tube barcode formats

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#1336]
**User story**
As a user I want to be able to create tubes in Sequencescape via manifest upload with barcodes in formats specified by the customers of the 007c project.

**Who are the primary contacts for this story**
Laura, Katy

**Who is the nominated tester for UAT**
Laura

**Acceptance criteria**
New foreign barcode formats added as follows:
- [ ] TBC
- [ ] ...

**Comment by Lesley84** [comment]
I will pick this question up at today's working group meeting. Apologies for the delay

**Comment by KatyTaylor** [comment]
This was confirmed (description updated) following working group meeting last week.

**Comment by KatyTaylor** [comment]
@Lesley84 Yes sort of, it will reject any barcodes that don't match one of the formats known to Sequencescape.
